Dockerfile-dev

```diff
@@ -30,4 +30,8 @@ RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSI
 
 RUN gem install decidim-dev:$decidim_version
 
+RUN cd /tmp && \
+    decidim decidim_app && \
```

Comment:
Missing `bundle install`?

Reply:
It's done behind the scenes by the generator.

Reply:
👍 Ah yes, I didn't see you were using the `--skip-bundle` option in the `Dockerfile-deploy` file

Reply:
I think maybe this warrants a comment since it's not obvious why we are generating an application and removing it right after.

Reply:
👍  to what @deivid-rodriguez says!